lib/states/wi/madison.R

```diff
@@ -52,31 +82,50 @@ clean <- function(d, helpers) {
     vehicle_year = Year,
     vehicle_color = Color,
     vehicle_registration_state = State,
-    posted_speed = Limit
+    posted_speed = Limit,
+    subject_age = Age, # new data has age, but for some reason, all are coerced to NA
+    time = Time
   ) %>%
   separate_cols(
     OfficerName = c("officer_last_name", "officer_first_name")
   ) %>%
   mutate(
-    # NOTE: Statute Descriptions are almost all vehicular, there are a few
+    # OLD NOTE: Statute Descriptions are almost all vehicular, there are a few
     # pedestrian related Statute Descriptions, but it's unclear whether
     # the pedestrian or vehicle is failing to yield, but this represents a
     # quarter of a percent maximum
+    # UPDATED NOTE: Similarly, there are a few pedestrian related Statute
+    # Descriptions, but they are extremely infrequent (~ < a quarter of a percent)
     type = "vehicular",
     speed = as.integer(posted_speed) + as.integer(OverLimit),
     date = parse_date(Date, "%Y/%m/%d"),
-    time = parse_time(Time, "%H:%M:%S"),
     location = coalesce(onStreet, onStreetName),
-    warning_issued = is.na(`Ticket #`),
-    citation_issued = !is.na(`Ticket #`),
+    # logic: for old data, if ticket # is NA, it's a warning
+    # for new data, if Ticketed == "Warning", it's a warning
+    # new data doesn't have ticket # var; old data doesn't have ticketed var
+    # so if Ticketed is NA and Ticket # is NA, it's a warning
+    # if Ticketed == "Warning", it's a warning
+    # if Ticketed == "ELCI", its a ticket
+    # if Ticket # is not NA, it's a ticket
+    warning_issued = case_when(
+      Ticketed=="Warning" ~ TRUE,
```

Comment:
I would add some spaces around the `==` here to make it easier to read. See: https://style.tidyverse.org/syntax.html#spacing.